src/messages/formatters.ts

```diff
@@ -1,3 +1,5 @@
+import type { URL } from 'url';
```

Comment:
`URL` is available as a global object

Reply:
Yes, yes it is. HOWEVER, with TS, the incompatibilities between the DOM URL and @types/node URL's cause issues at the global scope, hence the import. We can include the DOM typings and use the global, or stick to importing

Reply:
Oh, shouldn't the methods allow both the imported and global class then? Otherwise you currently wouldn't be able to pass an instance of the global class to these methods from what I understood

Reply:
Most users will use the built-in one, which is the one from url. Besides, if we support both, we may have issues later when the library uses a global URL they don't have.